filer/fields/folder.py

```diff
@@ -84,6 +79,8 @@ def obj_for_value(self, value):
             obj = self.rel.model._default_manager.get(**{key: value})
         except ObjectDoesNotExist:
             obj = None
+        except ValueError:
+            obj = None
```

Comment:
Hmmm, this feels a bit dangerous to me – you're catching a very generic error from a pretty deep call tree (the entire ORM machinery), which might suppress something that you don't want to suppress. I think it would be better to explicitly handle this case (with no value) before calling into the ORM.